site/docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ const config = {
   title: 'Very Good CLI',
   tagline:
     'A Command-Line Interface to generate scalable templates and use helpful commands.',
-  url: 'https://cli.vgv.dev',
+  url: 'https://verygoodopensource.github.io',
```

Comment:
Intentional?

Reply:
yeah we don't have a custom domain yet...

Reply:
we'll revert when we do

Reply:
we'll revert when we do

Reply:
yeah in order for the og images to work. we'll need to revert this when we configure the subdomain.